support/src/main/java/org/jenkinsci/plugins/workflow/support/steps/StageStepExecution.java

```diff
@@ -260,36 +274,38 @@ private static void cancel(StepContext context, StepContext newer) throws IOExce
     }
 
     private static final class Stage {
+        /** Maximum permitted size of {@link #holding}, or null for unbounded. */
+        @CheckForNull Integer concurrency;
```

Comment:
Please avoid reformatting existing code. It makes the PR diff too long and hard to review. The fewer diff lines, the better.